snippets/python/src/receive_onchain.py

```diff
@@ -7,6 +7,8 @@ def generate_receive_onchain_address(sdk_services):
 
     # Send your funds to the below bitcoin address
     address = swap_info.bitcoin_address
+    print("Minimum amount allowed to deposit in sats: {}", swap_info.min_allowed_deposit)
+    print("Maximum amount allowed to deposit in sats: {}", swap_info.max_allowed_deposit)
```

Comment:
I think you need to use `"{}".format(var)` with `{}`
```suggestion
    print("Minimum amount allowed to deposit in sats:", swap_info.min_allowed_deposit)
    print("Maximum amount allowed to deposit in sats:", swap_info.max_allowed_deposit)
```

Reply:
Done